### PR TITLE
Final Capstone - Booking-app

### PR DESCRIPTION
**Pull request by enis-memic**
# Booking-app
### [Frontend PR](https://github.com/ernestmusong/Booking-app-frontend/pull/15)

- [Project Requirements](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md)
- [Kanban board](https://github.com/enis-memic/booking_app_api/projects/2)

### Api docs
- [Api documentations](https://booking-app-api-lmvm.onrender.com/api-docs/index.html)
- You can also check it on _localhost:3000/api-docs_

### Frontend repository
- [Live demo](https://booking-app-frontend-0qon.onrender.com/)
- [Frontend repository](https://github.com/ernestmusong/Booking-app-frontend)

### Authors
- @enis-memic 
- @ernestmusong 
- @PowerLevel9000 
- @Habtamu210 

##### Important steps after installing project to local machine

1. Please create `.env` file and add this line SECRET_KEY_BASE = your secret key.
2. Generate secrey key by running command `rails secret`
3. Check Ruby version and database.yml file. ( Insert your username and password if there is required)
4. Please run `rails db:create` and `rails db:migrate`.
5. Please run `rails db:seed` for gathering all the cars and admin user.
6. Login for admin is: `(email: admin@admin.com ; password : password)`

##### To see app running locally

- Please switch base url in frontend app from rendered url to our localhost URL in order to see our app running locally.  `http://localhost:3000/`

## Happy Devloping 
CelebrationGIF

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+  rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  bin/rails rspec spec
+```
```
Comment (Meltrust) [inline README.md:188]: - [x]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @enis-memic,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Nice code organization ✔️ 
-  API is working well✔️ 
-  Good readme ✔️ 
-  Nice instructions ✔️ 



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: README.md (hunk 1 above)
Comment (enis-memic) [reply README.md:188]: Hello @Meltrust, thank you for requesting those changes. We will definitely update it!

**Review by Meltrust — APPROVED** [review]
#### Hi @enis-memic,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- Superfluous test folder has been removed✔️ 
- Nice work✔️ 




Cheers and Happy coding!👏👏👏


------

**Pull request merged**